### DISCUSSION TITLE
Documentation on Self Hosting is not uptodate

### DISCUSSION THREAD

**Discussion by madanyang** [plausible/analytics#3045] [Self-Hosted Support]
Hi,

Currently the documentation regarding [self hosting](https://plausible.io/docs/self-hosting) plausible is misinforming. Just for an example the requirements for the hardware says  the following which is not true as arm64 is available.

"The server must have a CPU with x86_64 architecture and support for SSE 4.2 instructions"

I am not sure if this is correct anymore regarding the use of maxmind geo location as the sample docker-compose.yml on the hosting does not mention anything like.
1. MaxMind geolocation database
```shell
$ docker-compose -f docker-compose.yml -f geoip/docker-compose.geoip.yml up -d
``` 

An updated version of documentation on what is required and what not along with up to date instructions would be much appreciated. Currently checking the discussions, looking at the docker-compose and the documentation is tiresome and confusing. 

Thanks in advance

**Comment by ruslandoga** [comment]
👋 @madanyang 

> "The server must have a CPU with x86_64 architecture and support for SSE 4.2 instructions"

I think this requirement comes from ClickHouse. Once we update to > 22.9 by default, it can be removed.

**Reply by ruslandoga** [reply]
I'm not sure what the problem is with MaxMind geolocation database. Could you please explain it in a bit more detail?

**Reply by ruslandoga** [reply]
Note that there also would be some updates to the docs once `v2` is released: https://github.com/plausible/docs/pull/399

I'd be happy if you could check it out and leave some comments on what could be further improved :)

**Reply by madanyang** [reply]
you mention in [upgrading to v2.0.0-rc.2](https://github.com/plausible/analytics/discussions/3011)  

Enable automatic MaxMind GeoLite2 updates (optional)

In your plausible-config.env set [MAXMIND_LICENSE_KEY](https://www.maxmind.com/en/accounts/current/license-key) environment variable and get an automatically updated [GeoLite2 City](https://dev.maxmind.com/geoip/geolite2-free-geolocation-data) geolocation database. The database edition is configurable with MAXMIND_EDITION environment variable and defaults to GeoLite2-City.

- IP_GEOLOCATION_DB=...
+ MAXMIND_LICENSE_KEY=LNpsJCCKPis6XvBP
+ MAXMIND_EDITION=GeoLite2-City

Where as the documentation page for [maxmind](https://plausible.io/docs/self-hosting#1-maxmind-geolocation-databasel) mentions use of geoip docker which is located in the hosting repo.
``
$ docker-compose -f docker-compose.yml -f geoip/docker-compose.geoip.yml up -d
``

So do you need the docker for geoip or not. From the comment you made in the upgrade version it is not clear. Which one is correct

**Reply by ruslandoga** [reply]
The release notes are "more" correct as they are more up to date. With v2 no extra containers are needed (and I think it says so in the notes, perhaps not in a clear enough way).

:)

**Reply by ruslandoga** [reply]
I'll add that change to the docs as well. Thank you for noticing it and telling me about it!
